notebooks/official/feature_store/online_feature_serving_bigquery_data_with_feature_store_2_0.ipynb

```diff
@@ -0,0 +1,1167 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.                string_filters=[ship_country_codes_filter, currency_filter],
</pre><p>[country_filter] instead of [ship_country_codes_filter, currency_filter]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>

Reply:
<p>Changed</p>